modules/auto-react/config.json

```diff
@@ -29,6 +29,20 @@
         "value": "string"
       }
     },
+    {
+      "field_name": "membersReply",
+      "default": {},
+      "humanname-de": "Erwähnungsantworten",
+      "humanname-en": "Mention-replys",
+      "type": "keyed",
+      "description-en": "Here you can add members and the reply on mentions of them",
+      "description-de": "Du kannst hier NutzerIDs und die Antwort auf Erwähnungen dieser eintragen",
+      "allowEmbed": true,
```

Comment:
This is not supported, please use config-elements instead

Reply:
ok. I have no idea how to use this 😅 I can create a working config with it, but how do I read from it?

Reply:
Same way as usual files, but you get an array of objects with the configured values. 

Tipp: you can always use `console.log(value)` to see the content of a variable